docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
+against the original ones. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires and 
+allow us to express circuits
+with less differing vectors 
+per polynomial. 
+
+PLONK uses an argument to check 
+the correctness of a shuffle in 
+its proving algorithm. This allows 
+for the checking of individual 
+wires rather than entire sets. 
+In principle, PLONK can compare sets 
+of polynomials for equality by 
+comparing an `n` degree polynomial 
+with the same `n` degree polynomial 
+where some elements have been 
+rotated. 
+
+Given a set of wire values, at a 
+particular position, we can create 
+a mapping for these wires into a 
+new position with a permuatation. 
+
+For example: 
+Given initial wire values, 
+1,2,3,4,5 each occupying 
+their respective positions
+A,B,C,D,E
+
+\begin{aligned}
+
+// +-----+-----+-----+-----+-----+
+// |  A  |  B  |  C  |  D  |  E  |
+// +-----+-----+-----+-----+-----+
+// |  1  |  2  |  3  |  4  |  5  |
+// +-----+-----+-----+-----+-----+
+
+\end{aligned}
+
+If we then have a mapping of:
+
+\begin{aligned}
+
+\\[
+\mathbf{a}(1) = 5
+\mathbf{a}(2) = 2
+\mathbf{a}(3) = 3
+\mathbf{a}(4) = 1
+\mathbf{a}(5) = 4
+\\]
+
+\end{aligned}
+
+Which gives a second, permutuated row, 
+of:
+
+\begin{aligned}
+
+// +-----+-----+-----+-----+-----+
+// |  A  |  B  |  C  |  D  |  E  |
+// +-----+-----+-----+-----+-----+
+// |  1  |  2  |  3  |  4  |  5  |
+// +-----+-----+-----+-----+-----+
+// |  3  |  5  |  1  |  4  |  2  |
+// +-----+-----+-----+-----+-----+
+
+\end{aligned}
+
+We can form equations of polynomials 
+to compare the wire values at each 
+position. As it possible to falsify
+proofs, by having the sum of the 
+equation arguments equal - without 
+having the identities between the
+monomials hold - therefore, two random 
+constants are introduced to the 
```

Comment:
I'm not sure the random challenges should be called constants.

One reason is that they change for every proof, and we have actual constants which do not.